docs/General/Manuals/client-manual.rst

```diff
@@ -404,6 +404,91 @@ consists of the following items:
 Game Menu
 ^^^^^^^^^
 
+The :guilabel:`Game` menu is used to conduct global level options on the client or the game being played.
+
+The :guilabel:`Game` menu has the following options:
+
+Save Game
+    Saves the game as it is :strong:`right now` to the ``saves`` directory in the user's profile directory.
+    On Linux and Mac OS, the profile directory is inside the User's home directory such as
+    ``~/.local/share/freeciv21``. On Windows the profile directory is inside the User ``AppData`` directory
+    such as ``C:\\Users\\<userName>\\AppData\\Roaming\\freeciv21``.
+
+Save Game As ...
+    Saves the game as it is :strong:`right now` to a location of your choosing.
+
+Save Map to Image
+    Save a ``.png`` image file of the map to the user's profile directory.
```

Comment:
Is there a better way to phrase this  ? Profile directory sounds too technical and requires further explanation (like a section which is linked to here)